charts/default-network-policies/templates/adminnetworkpolicy.yaml

```diff
@@ -0,0 +1,228 @@
+  ingress:
+    - name: allow-ingress-from-monitoring
+      action: Allow
+      from:
+        - namespaces:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - openshift-monitoring
+                  - openshift-user-workload-monitoring
+    - name: allow-ingress-from-konnectivity
+      action: Allow
+      from:
+        - pods:
+            namespaceSelector:
+              matchExpressions:
+                - key: kubernetes.io/metadata.name
+                  operator: In
+                  values:
+                    - kube-system
+            podSelector:
+              matchExpressions:
+                - key: app
+                  operator: In
+                  values:
+                    - konnectivity-agent
+    - name: allow-ingress-from-openshift-storage
+      action: Allow
+      from:
+        - pods:
+            namespaceSelector:
+              matchExpressions:
+                - key: kubernetes.io/metadata.name
+                  operator: In
+                  values:
+                    - openshift-storage
+            podSelector:
+              matchExpressions:
+                - key: app.kubernetes.io/name
+                  operator: In
+                  values:
+                    - csi-addons
+    - name: allow-ingress-from-openshift-operators
+      action: Allow
+      from:
+        - namespaces:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - openshift-operators
+    - name: allow-ingress-from-crossplane-namespace
+      action: Allow
+      from:
+        - namespaces:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - {{ .Values.crossplaneNamespace }}
+    - name: pass-ingress-from-application-namespaces
+      action: Pass
+      from:
+        - namespaces:
+            matchExpressions:
+              - key: apc.namespace.type
+                operator: In
+                values:
+                  - application
+    - name: pass-ingress-from-platform-namespaces
+      action: Pass
+      from:
+        - namespaces:
+            matchExpressions:
+              - key: apc.namespace.type
+                operator: In
+                values:
+                  - platform
+  egress:
+    - name: allow-egress-inside-cluster
+      action: Allow
+      to:
+      - namespaces: {}
+      - nodes: {}
+    {{- if .Values.proxyIPs }}
+    - name: pass-egress-to-proxy
+      action: Pass
+      to:
+      - networks:
+        {{- toYaml .Values.proxyIPs | nindent 8 }}
+    {{- end }}
+    - name: allow-all-private-cidrs-for-cluster-external-traffic
+      action: Allow
+      to:
+      - networks:
+        - 10.0.0.0/8
+        - 172.16.0.0/12
+        - 192.168.0.0/16
+---
```

Comment:
@pstefka , according to https://helm.sh/docs/chart_best_practices/templates/ 
- "Each resource definition should be in its own template file."
Suggest to split into multiple templates.